example/cmesh/t8_cmesh_read_from_vtk.cxx

```diff
@@ -0,0 +1,155 @@
+/*
+This file is part of t8code.
+t8code is a C library to manage a collection (a forest) of multiple
+connected adaptive space-trees of general element classes in parallel.
+
+Copyright (C) 2015 the developers
+
+t8code is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+
+t8code is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with t8code; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*/
+
+#include <t8_cmesh_vtk_reader.hxx>
+#include <t8_cmesh_vtk_writer.h>
+#include <t8_cmesh.h>
+#include <sc_options.h>
+#include <t8.h>
+#include <t8_vtk.h>
+#include <t8_forest.h>
+#include <t8_schemes/t8_default/t8_default_cxx.hxx>
+#include <t8_forest.h>
+
+/**
+ * Construct a cmesh read from a VTK-file type supported by our vtk-reader.
+ * Given the number of cell-values in the file we read the data again from the forest
+ * and write it into the file forest.(p)vtu . 
+ * 
+ * \param[in] prefix  The prefix of the file to read the mesh from
+ * \param[in] comm    The communicator used in this example
+ * \param[in] values_per_cell   The number of values per cell in the mesh.
+ */
+void
+t8_forest_construct_from_vtk (const char *prefix, sc_MPI_Comm comm,
+                              const int values_per_cell)
+{
+  /* Read a poly-data file (.ply, .vtp, .obj, .stl, .vtk, .g) and construct a cmesh 
+   * representing the mesh. If  there is any cell-data, it will be read too. 
+   * Triangle-strips and polygons will be broken down to multiple triangles. */
+  t8_cmesh_t          cmesh = t8_cmesh_read_from_vtk_poly (prefix, comm);
```

Comment:
can we add a command line parameter to control whether we use the `poly` or `unstructured` version?